src/components/NavigationBar.vue

```diff
@@ -22,43 +22,43 @@
 
 <style scoped>
 .navbar {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  z-index: 1;
   margin: 0 auto;
   padding: 1rem;
-  z-index: 1;
   width: 100%;
-  background-color: #333;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  background-color: var(--global-color-neutral-100);
   animation: var(--global-animation-fade-down);
 }
 
 .navbar-container {
-  display: flex;
   flex-wrap: wrap;
   justify-content: space-between;
   align-items: center;
   gap: calc(var(--global-layout-gutter) / 2);
+  display: flex;
   margin: 0 auto;
   max-width: var(--global-layout-size);
```

Comment:
Just also wanna point out that global vars can be locally overwritten if the need arises. Good tool to have.